docs/OMZETSTAPPEN.md

```diff
@@ -0,0 +1,29 @@
+# Omzetstappen veg2hab
```

Comment:
Deze file is nog niet af. Dit is ook wel een mooie plek om de argumenten voor de functies te omschrijven.

Reply:
Ja klopt, die heb ik alvast aangemaakt, om een paar kleine updates naar aanleiding van de gebruikerstest in te zetten. Wat mij betreft wordt dit aanvullen een nieuwe ticket